samples/cpp/whisper_speech_recognition/audio_utils.hpp

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2023-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <iostream>
+#include <vector>
+
+namespace utils {
+namespace audio {
+bool read_wav(const std::string& fname,
+              std::vector<float>& pcmf32,
+              std::vector<std::vector<float>>& pcmf32s,
```

Comment:
Return a pair of vectors instead. You can unpack the result in the sample with `auto [raw_speech, raw_speech_s]`